datalake/src/test/java/com/sequenceiq/datalake/service/sdx/stop/SdxStopServiceTest.java

```diff
@@ -78,18 +89,40 @@ public class SdxStopServiceTest {
     @Mock
     private RegionAwareInternalCrnGenerator regionAwareInternalCrnGenerator;
 
+    @Mock
+    private FlowChainLogService flowChainLogService;
+
+    @Mock
+    private FlowLogService flowLogService;
+
     @InjectMocks
     private SdxStopService underTest;
 
     @Test
     public void testTriggerStop() {
+        when(flowLogService.getLastFlowLog(anyLong())).thenReturn(Optional.empty());
         SdxCluster sdxCluster = sdxCluster();
-
         underTest.triggerStopIfClusterNotStopped(sdxCluster);
-
         verify(sdxReactorFlowManager).triggerSdxStopFlow(sdxCluster);
     }
 
+    @Test
```

Comment:
Please add a test to make sure it will allow stop operation if for example the resize flow was already completed.